app/presenters/tree_builder_report_dashboards.rb

```diff
@@ -19,7 +19,7 @@ def set_locals_for_render
   # Get root nodes count/array for explorer tree
   def x_get_tree_roots(count_only, _options)
     objects = []
-    default_ws = MiqWidgetSet.find_by_name_and_read_only('default', true)
+    default_ws = MiqWidgetSet.where(:name => 'default', :read_only => true).first
```

Comment:
For `find_by_*` as opposed to `find_all_by_*` prefer `MiqWidgetSet.find_by(:name => 'default', :read_only => true)` vs `.where().first`